### PR TITLE
ROX-17465: Add RHACS Workload CVE info

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- merge to `rhacs-docs`
- cherry pick to `rhacs-docs-4.1`

[Issue](https://issues.redhat.com/browse/ROX-17465)

Link to docs preview:
- [Viewing workload CVEs in Vulnerability Management (2.0)](https://61387--docspreview.netlify.app/openshift-acs/latest/operating/manage-vulnerabilities/vulnerability-management.html#vulnerability-management20-view-workload-cve_acs-operating-manage-vulnerabilities)
- Note/cross-reference & "Additional reference" link added to the following sections (**other changes to this content outside of the scope of this PR - not updating this content as it is deprecated**):
	- https://61387--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html#view-images-in-your-environment_examine-images-for-vulnerabilities
	- https://61387--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html#viewing-dockerfile-for-image_examine-images-for-vulnerabilities
	- https://61387--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html#identify-container-image-layer-that-introduces-vulnerabilities_examine-images-for-vulnerabilities
	- https://61387--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html#identify-dockerfile-line-component-cve_examine-images-for-vulnerabilities
	- https://61387--docspreview.netlify.app/openshift-acs/latest/operating/examine-images-for-vulnerabilities.html#identify-operating-system-of-the-base-image_examine-images-for-vulnerabilities

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by md2119 — APPROVED** [review]

File: modules/vulnerability-management20-view-cve.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Namespace
+* Cluster
+. Optional: Use the *CVE severity* drop-down list to select the severities of the CVEs that you want to display.
+. Click the desired button to view a list of vulnerabilities, images, or deployments in the system.
++
+[NOTE]
+====
+The *Filtered view* icon indicates that the results displayed have been filtered based on the criteria that you selected. You can click *Clear filters* to remove all filters, or remove individual ones by clicking on them.
+====
+. Click on a CVE, image name, or deployment name in the list of results to view more information about the item. For example, depending on the item type, you can view information such as if a CVE is fixable, if an image is active, the Dockerfile line in the image that contains the CVE, and external links to information about the CVE in Red Hat and other CVE databases.
```
Comment (lahinson) [inline modules/vulnerability-management20-view-cve.adoc:29]:
```suggestion
. In the list of results, click a CVE, image name, or deployment name to view more information about the item. For example, depending on the item type, you can view the following information:
+
* Whether a CVE is fixable
* Whether an image is active
* The Dockerfile line in the image that contains the CVE
* External links to information about the CVE in Red Hat and other CVE databases
```

**Review by lahinson — COMMENTED** [review]
Nice work, @kcarmichael08! I included a few suggestions, as well as a few pointers to guidance in IBM Style and the SSG for your consideration.

File: modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc [hunk 2]
```diff
@@ -15,3 +15,5 @@ Use the *Vulnerability Management* view to identify vulnerable components and th
 . Select an image from either the *Top Riskiest Images* widget or click the *Images* button at the top of the Dashboard and select an image.
 . In the *Image* details view, next to *Dockerfile*, select the expand icon to see a summary of image components.
 . Select the expand icon for specific components to get more details about the CVEs affecting the selected component.
+
+You can also view this information by navigating to *Vulnerability Management (2.0)* -> *Workload CVEs*. See "Viewing workload CVEs in Vulnerability Management (2.0)" in the "Additional Resources" section for more information.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc:19]: I did see this guidance, but we already have 180+ instances of "navigate" in the ACS docs. Maybe we can do a full-scale change at some point, but I was keeping this language for consistency.

File: modules/view-images-in-your-environment.adoc [hunk 3]
```diff
@@ -6,8 +6,11 @@
...
 
 [role="_abstract"]
-With {product-title} you can view the details for all container images in your clusters.
+With {product-title-short}, you can view the details for all container images in your clusters.
 
 .Procedure
-. Navigate to the RHACS portal and click *Vulnerability Management* from the left-hand navigation menu.
-. To view details for all the images in your cluster, select *Images* on the *Vulnerability Management* view header.
+. In the {product-title-short} portal, navigate to *Vulnerability Management* -> *Dashboard*.
+. To view details for all the images in your cluster, select *Images* in the *Vulnerability Management* view header.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/view-images-in-your-environment.adoc:13]: I wasn't making changes to the previous content (this section was updated to add a missing item in the previous step), but I will fix.

File: modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc:19]: @kcarmichael08 Thanks for letting me know! I had actually wondered whether "navigate" was standard in the ACS docs after I saw it so much. I totally understand about keeping it for consistency.

File: modules/vulnerability-management20-view-cve.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+[role="_abstract"]
+You can view a comprehensive list of vulnerabilities, or CVEs, in {product-title-short} across images and deployments. You can use the search filter bar to select specific CVEs, images, deployments, namespaces, or clusters.
+
+:FeatureName: Vulnerability Management 2.0
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. In the {product-title-short} portal, navigate to *Vulnerability Management 2.0* -> *Workload CVEs*.
+. From the drop-down list, select the search criteria you want to use. You can select an item type, such as a cluster, from the drop-down list, and then select the specific name of the item. You can add additional items to the filter by selecting another item from the drop-down list and selecting the specific name of the new item. For example, you can select a specific image and a specific cluster to limit results to those selections. You can filter on the following items:
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/vulnerability-management20-view-cve.adoc:16]: No label. I'll add a pic below. You choose either Deployment, CVE, Image, Namespace, or cluster from the list.

**Comment by kcarmichael08** [comment]
Screenshot 2023-06-21 at 12 09 00 PM

File: modules/vulnerability-management20-view-cve.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/vulnerability-management20-view-cve.adoc:16]: In that case, I think I'd keep "drop-down list" for the first mention, but just "list" in subsequent mentions, assuming that within the context, the reader will know which list you're referring to.

File: modules/view-images-in-your-environment.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/view-images-in-your-environment.adoc:13]: > From [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps): "If you orient the user to the location where the action is performed or explain the purpose of the action, do so before describing the action."

I'm explaining the purpose of the action first here, correct? (To view details for all images in the cluster....")

[the remark this answers is not in the document]
Comment (lahinson) [reply modules/view-images-in-your-environment.adoc:13]: Ack, sorry for adding extra work to your plate! I didn't realize that the content hadn't been changed. And yes, in the revision, you're explaining the purpose of the action first.

File: modules/vulnerability-management20-view-cve.adoc [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* CVE
+* Image
+* Deployment
+* Namespace
+* Cluster
+. Optional: Use the *CVE severity* drop-down list to select the severities of the CVEs that you want to display.
+. Click the desired button to view a list of vulnerabilities, images, or deployments in the system.
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/vulnerability-management20-view-cve.adoc:23]: Relevant is great, I was drawing a blank on this!

File: modules/view-images-in-your-environment.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/view-images-in-your-environment.adoc:13]: No worries at all, I really appreciate the feedback!

**Pull request merged**